viewing_party/party.py

```diff
@@ -1,23 +1,213 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from symbol import single_input
+
+
+def create_movie(title, genre, rating): #Wave 1, step 1 done
+    movie_dict={}
+    if title is None or genre is None or rating is None:
+        return None
+
+    movie_dict["title"]=title
+    movie_dict["genre"]=genre
+    movie_dict["rating"]=rating

```

Comment:
this works just fine! We could also do this all at once: 
```suggestion
    movie_dict = {
        "title": title
        "genre": genre
        "rating": rating
    }

```
Now, obviously this isn't one line like this, so we aren't saving space. But we can do it all in one expression instead of 4.